Entity/Entity.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
Remove this using statement since we're not using threads (there is an automated way to do this I'll show you later on)